packages/core/src/index.ts

```diff
@@ -850,19 +894,33 @@ class Hopr extends EventEmitter {
 
   // This is a utility method to wait until the node is funded.
   public async waitForFunds(): Promise<void> {
-    return new Promise((resolve) => {
-      const tick = () => {
-        this.getNativeBalance().then((nativeBalance) => {
-          if (nativeBalance.toBN().gt(MIN_NATIVE_BALANCE)) {
-            resolve()
-          } else {
-            log('still unfunded, trying again soon')
-            setTimeout(tick, CHECK_TIMEOUT)
-          }
-        })
-      }
-      tick()
-    })
+    const MIN_DELAY = durations.seconds(30)
+    const MAX_DELAY = durations.seconds(200)
```

Comment:
Do we check for funds after 200 seconds?

Reply:
we stop if people _never_ funded their nodes after 1 hour

Reply:
we are working on a different PR a way to allow users to re-start their node, currently this will conflict with other work Peter is working on